src/scss/components/_pagination.scss

```diff
@@ -16,16 +16,21 @@
     display: inline-block;
 
     & + & {
-      margin-left: 3px;
+      margin-left: 2px;
     }
 
     a {
       display: block;
-      padding: 6px 10px;
+      padding: 4px 9px;
       width: 100%;
       height: 100%;
-      border: solid 1px get-color((color: blackNeutral, opacity: .2));
+      border: solid 1px get-color((color: rastaGray, opacity: .2));
       border-radius: $borderRadius;
+      transition-duration: .2s, .2s, .2s;
```

Comment:
@iondrimba I prefer 200ms instead of .2s, I think it's more readable. What do you think?